### PR TITLE
Blog Post: Introducing the aNCA package for Pharmaverse!

### LINKED ISSUE #233: Blog Post:  Introducing the aNCA package for Pharmaverse!
### Blog Post

I want to make a post to introduce the new aNCA pharmaverse package in development.

### PR DESCRIPTION
**Pull request by Gero1999**
Closes #233 
 
 Hope everything looks good for the post!😺  

# To check
- [X] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update), and make sure the corresponding issue is linked in the Development section on the right hand side
- [X] Ensure your new post folder is of the form `"posts/zzz_DO_NOT_EDIT_<your post title>"`. This is so that the post date can be auto-updated upon the merge into `main`.
- [X] Run the script from `CICD.R` line by line to first check the spelling in your post and then to make sure your code is compatible with our code-style. Address any incongruences by following the instructions in the file! 
- [X] Choose (possibly several) `tag(s)` or categories from the current list: ` c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")` for your blog post. If you cannot find anything that fits your blog post, propose a new tag to the maintainers! Note: if you use a tag not from this list, the "Check Post Tags" CICD pipeline will error. We occasionally tidy up all `tags` for consistency.
- [X] Add a short description for your blog post in the `description` field at the top of the markdown document.
- [X] Blog post is *short*, *personalized*, *reproducible* and *readable*
- [X] Add a disclaimer at the top of your post, if appropriate (example: **Disclaimer** 
This blog contains opinions that are of the authors alone and do not necessarily reflect the strategy of their respective organizations.)
- [x] Address all merge conflicts and resolve appropriately
- [x] Assign two of us (@bms63, @manciniedoardo, @StefanThoma, @kaz462) as reviewers in the PR.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: posts/zzz_DO_NOT_EDIT_a_nca/a_nca.qmd [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### **The aNCA package empowers users to:**
+
+-   Perform NCA on both non-clinical and clinical data
+-   Automatically calculate PK parameters for various dosing regimens
+-   Conduct manual selections and exclusions for lambda-z regression
+-   Visualize results dynamically with editable plots and summary tables
+-   Export PK parameters into a comprehensive, ready-to-use reports
+
+This tool is built in `RShiny` with user-friendliness in mind, allowing both novice and experienced users to seamlessly integrate it into their workflows.
```
[the remark this answers is not in the document]
Comment (bms63) [reply posts/zzz_DO_NOT_EDIT_a_nca/a_nca.qmd:38]: can you also provide link to shiny package

**Review by bms63 — CHANGES_REQUESTED** [review]
Is the App far enough along to have a simple gif included in the post.

Just to bring some excitement!!!  Lovely summary

File: posts/zzz_DO_NOT_EDIT_a_nca/a_nca.qmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (Gero1999) [reply posts/zzz_DO_NOT_EDIT_a_nca/a_nca.qmd]: Hello @bms63 ! I provided the GitHub repo, sounds good enough? 

https://github.com/pharmaverse/aNCA

**Comment by bms63** [comment]
> > p far enough along to have a simple gif included in the post.
> > Just to bring some excitement!!! Lovely summary
> 
> We though about it and I think is a great idea and thanks!
> 
> One quest, this is our first time publishing, is it ok if we make a future edition of the repo to include a video/gif when we have something final tailored and open a new pr?

Yes please keep the post updated.  You can either make a new post and reference the old or just update the same post.  Up to you. 

We actually have it set up so if there is package code from a year ago and a new version of the package is published with breaking changes, then the post fails to build.  Kind of a headache, but also means when someone comes to look at the post the code is most up to date!!

**Comment by Gero1999** [comment]
hello @bms63 @manciniedoardo I solved the merge conflicts left and applied the changes requested. Let me know if all looks good or if I need to do additional modifications before proceeding with the merge! 🙌🏽

**Review by manciniedoardo — APPROVED** [review]
LGTM thanks! @bms63 feel free to approve and merge when ready next week.

**Pull request merged**